### PR TITLE
Update 0002-use-github-for-documentation.md

### PR DESCRIPTION
**Pull request by oerc0122**
Update as suggested by #457

### CODE REVIEW

File: documentation/adr/0002-use-github-for-documentation.md [hunk 1]
```diff
@@ -16,14 +18,21 @@ Documentation will directly reference Horace and Herbert projects.
 
 The GitHub Wiki repository does not support pull requests, or the display of any content not on the the `master` branch.
 
+
 ## Decision
 
 We will store documentation in the `Horace/documentation` folder and link to this from the the [GitHub Wiki](https://github.com/pace-neutrons/pace-developers/wiki) which is attached to the PACE project on GitHub.
```
Comment (cmarooney-stfc) [inline documentation/adr/0002-use-github-for-documentation.md:24]: @oerc0122 Presumably the folder in this project, so add "folder in the Horace Github repository."

File: documentation/adr/0002-use-github-for-documentation.md [hunk 2]
```diff
@@ -16,14 +18,21 @@ Documentation will directly reference Horace and Herbert projects.
...
+
 ## Decision
 
 We will store documentation in the `Horace/documentation` folder and link to this from the the [GitHub Wiki](https://github.com/pace-neutrons/pace-developers/wiki) which is attached to the PACE project on GitHub. 
 
+The documentation will be supported and built by the Jenkins pipeline and automatically deployed on:
+- nightly builds (unstable)
+- release pipeline (stable & horace-version)
+
+The Jenkins pipeline will also build the documentation as a zipped artifact for examination on other builds.
```
Comment (cmarooney-stfc) [inline documentation/adr/0002-use-github-for-documentation.md:30]: @oerc0122 Not clear what examination on other builds means. What workflow would this imply? I was presuming that zipped artefacts could be unzipped at remote locations for easy access by users also downloading Horace/PACE.

File: documentation/adr/0002-use-github-for-documentation.md [hunk 3]
```diff
@@ -16,14 +18,21 @@ Documentation will directly reference Horace and Herbert projects.
...
+
 ## Consequences
 
 - Documentation will be written in markdown.
 - Storing in the main source repository means changes can be tracked and reviewed through pull requests. 
-- Pdf versions of the documentation can be quickly and simply generated using some off-the-shelf tooling if required.
+- PDF versions of the documentation can be quickly and simply generated using some off-the-shelf tooling if required.
 - Markdown stored at [pace-developers/wiki](https://github.com/pace-neutrons/pace-developers.wiki.git) will link to the documentation.
 - Documentation will be stored as part of the Horace project [Horace/documentation](https://github.com/pace-neutrons/horace.git).
```
Comment (cmarooney-stfc) [inline documentation/adr/0002-use-github-for-documentation.md:38]: This link could also be used to deal with my comment on line 24.

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

File: documentation/adr/0002-use-github-for-documentation.md [hunk 4]
```diff
@@ -16,14 +18,23 @@ Documentation will directly reference Horace and Herbert projects.
...
 ## Decision
 
-We will store documentation in the `Horace/documentation` folder and link to this from the the [GitHub Wiki](https://github.com/pace-neutrons/pace-developers/wiki) which is attached to the PACE project on GitHub. 
+~We will store documentation in the `Horace/documentation` folder and link to this from the [GitHub Wiki](https://github.com/pace-neutrons/pace-developers/wiki) which is attached to the PACE project on GitHub.~ 
+
+We will store documentation in the `Horace/documentation` folder which will be built regularly and deployed to the GitHub pages under the following scheme: 
+
+The documentation will be supported and built by the Jenkins pipeline and automatically deployed on:
+- nightly builds (unstable)
+- release pipeline (stable & horace-version)
```
Comment (hsautessella) [inline documentation/adr/0002-use-github-for-documentation.md:30]: The release pipelines build release packages, but we don't necessarily release those packages (we do some testing first). Should the Deploy pipeline or some GitHub action deploy releases?

File: documentation/adr/0002-use-github-for-documentation.md [hunk 5]
```diff
@@ -16,14 +18,23 @@ Documentation will directly reference Horace and Herbert projects.
...
+
+The Jenkins will also build the documentation as a zipped artefact on other pipelines.
```
Comment (hsautessella) [inline documentation/adr/0002-use-github-for-documentation.md:32]: Perhaps we should add what other pipelines and _why_ they will build the docs (to check that a code/documentation change has not broken the docs build?).

**Review by hsautessella — CHANGES_REQUESTED** [review]
Looks good. Just a couple of things

**Review by hsautessella — APPROVED** [review]
👍

**Review by cmarooney-stfc — APPROVED** [review]
Just noting that the rich documentation / source documentation toggle top right toggles between a diff view and a track changes view typora style - so it is possible to see the rendered document from the file changes tab!

**Pull request merged**